doc/tools/polynomial.rst

```diff
@@ -0,0 +1,205 @@
+:orphan:
```

Comment:
Where does this get built as part of the documentation? I built the docs, and don't see it anywhere...